src/jaxoplanet/_src/experimental/starry/rotation.py

```diff
@@ -0,0 +1,141 @@
+from functools import partial
+from typing import Callable
+
+import jax
+import jax.numpy as jnp
+import numpy as np
+from jax.scipy.linalg import block_diag
+from scipy.special import factorial
+
+from jaxoplanet._src.types import Array
+
+
+@jax.jit
+def axis_to_euler(u1: float, u2: float, u3: float, theta: float):
+    """Axis-angle rotation matrix"""
```

Comment:
Could you expand this docstring a bit? I know that the other `starry` matrices don't have good docstrings, but let's start with this one and we can update the others in the next PR.